src/components/Projects.tsx

```diff
@@ -0,0 +1,192 @@
+'use client';
+import React, { useState } from 'react';
+import { ProjectPageType } from '../pages';
+import {
+  Container,
+  Heading,
+  Text,
+  VStack,
+  Link,
+  Flex,
+  WrapItem,
+  Image,
+  IconButton,
+  useMediaQuery,
+} from '@chakra-ui/react';
+import {
+  FaArrowRightLong,
+  FaCircleChevronLeft,
+  FaCircleChevronRight,
+} from 'react-icons/fa6';
+// Import Swiper styles
+import 'swiper/css';
+import { Swiper, SwiperSlide, useSwiper } from 'swiper/react';
+
+// Types for SwiperNavButtons props
+type SwiperNavButtonsProps = {
+  handleButtonAction: (direction: 'left' | 'right') => void;
+  currentIndex: number;
+  maxIndex: number;
+};
+
+const SwiperNavButtons: React.FC<SwiperNavButtonsProps> = ({
+  handleButtonAction,
+  currentIndex,
+  maxIndex,
+}) => {
+  const swiper = useSwiper();
+
+  return (
+    <Flex gap={4} justifyContent={'center'} mt={4}>
+      <IconButton
```

Comment:
Icon Color should have a `dark` mode bg.

Reply:
icon does not have a `bg` in both scenario. The default `bg` is set to transparent as it is not required in both dark and light mode.